packages/platform-core/src/cdk/enum_converters.ts

```diff
@@ -95,3 +100,27 @@ export class LogLevelConverter {
     }
   };
 }
+
+/**
+ * Converts DataLogLevel to CDK types.
+ */
+export class DataLogLevelConverter {
```

Comment:
This converter can (and should be) moved to `backend-data`. There's no need for it to be visible (other converters must be accessible from 2 packages).

It can be then package-private (not exposed).

Reply:
There's one more alternative to try here.

1. The `DataLogLevelConverter` is removed.
2. The existing `LogLevelConverter` gets `LogLevelConverter.toCDKFieldLogLevel` method - handle subset, throw otherwise.
3. The `DataLogLevel` is moved to `backend-data`
4. `backend-data` calls `LogLevelConverter.toCDKFieldLogLevel` with `DataLogLevel` which works because `DataLogLevel` it's a subset.

The caveat is that we expose public api just for consistency sake. 

@Amplifiyer wdyt?

Reply:
We are already doing that with `toCDKLambdaApplicationLogLevel`, so this suggestion is definitely in line with that.

Reply:
`LogLevelConverter.toCDKAppsyncFieldLogLevel(logLevel: LogLevel)` (we prefix  `ApplicationLogLevel` with `Lambda`).

Reply:
For this to work, we will need to expand `LogLevel`
https://github.com/aws-amplify/amplify-backend/blob/6dd93d17fd04df3bcc2984682a2aa6b383c9a445/packages/plugin-types/src/log_level.ts#L1
to
```
export type LogLevel = 'info' | 'debug' | 'warn' | 'error' | 'fatal' | 'trace' | 'none' | 'all';
```
and change `DataLogLevel` and `FunctionLogLevel`:
```
export type DataLogLevel = Extract<LogLevel, 'none' | 'all' | 'info' | 'debug' | 'error'>;
```
```
export type FunctionLogLevel = Extract<LogLevel, 'info' | 'debug' | 'warn' | 'error' | 'fatal' | 'trace'>;
```

Is that what you are suggesting?

Reply:
Yes. I did not notice that appsync has one extra value.

We should also cover this line in this case https://github.com/aws-amplify/amplify-backend/blob/e7f415cee5d84f4494538c2583af62a058707cdb/packages/backend-ai/src/conversation/factory.ts#L162 . (same transformation like in functions case).

Reply:
updated `ConversationHandlerFunctionLogLevel`